ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionDetailsCell.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+import { Td } from '@patternfly/react-table';
+
+import LinkShim from 'Components/PatternFly/LinkShim';
+import { exceptionManagementPath } from 'routePaths';
+import { getUrlQueryStringForSearchFilter } from 'utils/searchUtils';
+import { VulnerabilityState } from 'types/cve.proto';
+import { ensureExhaustive } from 'utils/type.utils';
+
+function getExceptionManagementURL(
+    cve: string,
+    vulnerabilityState: Exclude<VulnerabilityState, 'OBSERVED'>
+): string {
+    const query = getUrlQueryStringForSearchFilter({ cve });
+
+    switch (vulnerabilityState) {
+        case 'DEFERRED':
+            return `${exceptionManagementPath}/approved-deferrals?${query}`;
+        case 'FALSE_POSITIVE':
+            return `${exceptionManagementPath}/approved-false-positives?${query}`;
+        default:
```

Comment:
If it was `DENIED`, the CVEs would once again be shown in the "Observed" tab, which doesn't display this link cell. If one of those CVEs was part of another approved request, it would show up under either Deferrals of False positives.